plugin.video.yelo/addon.xml

```diff
@@ -26,10 +26,21 @@
         <platform>all</platform>
         <license>GNU General Public License, v3</license>
         <email>shycoderX@protonmail.com</email>
-        <source>https://github.com/shycoderX/repo-plugins/tree/leia/plugin.video.yelo</source>
+        <source>https://github.com/add-ons/plugin.video.yelo</source>
```

Comment:
Can you change the license to a spdx compatible identifier? Should be GPL-3.0-only